vapi/esx/settings/clusters/vms/transition.go

```diff
@@ -0,0 +1,226 @@
+// © Broadcom. All Rights Reserved.
+// The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+// SPDX-License-Identifier: Apache-2.0
+
+package vms
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/vmware/govmomi/vapi/cis/tasks"
+	"github.com/vmware/govmomi/vapi/esx/settings/clusters"
+	"github.com/vmware/govmomi/vapi/rest"
+	"github.com/vmware/govmomi/vim25/types"
+)
+
+type clusterTransitionPath types.ManagedObjectReference
+
+const (
+	TransitionPath = clusters.BasePath + "/%s/vms/transition"
```

Comment:
All exported symbols should have valid GoDoc comments.

Reply:
ok